python/test/test_decode_encode.py

```diff
@@ -502,32 +505,33 @@ def test_flat_binary_log_decode_bestpos(helper):
     assert log_header.receiver_sw_version == 32768
 
     # SOL_COMPUTED SINGLE 51.15043711386 -114.03067767000 1097.2099 -17.0000 WGS84 0.9038 0.8534 1.7480 \"\" 0.000 0.000 35 30 30 30 00 06 39 33\r\n"
-    assert bestpos.solution_status == ne.enums.SolStatus.SOL_COMPUTED
-    assert bestpos.position_type == ne.enums.SolType.SINGLE
-    assert bestpos.latitude == 51.15043711386
-    assert bestpos.longitude == -114.03067767000
-    assert bestpos.orthometric_height == 1097.2099
-    assert bestpos.undulation == -17.0000
-    assert bestpos.datum_id == ne.enums.Datum.WGS84
-    assert bestpos.latitude_std_dev == approx(0.9038, abs=1e-5)
-    assert bestpos.longitude_std_dev == approx(0.8534, abs=1e-5)
-    assert bestpos.height_std_dev == approx(1.7480, abs=1e-5)
-    # assert bestpos.base_id == ""
-    assert bestpos.diff_age == 0.000
-    assert bestpos.solution_age == 0.000
-    assert bestpos.num_svs == 35
-    assert bestpos.num_soln_svs == 30
-    assert bestpos.num_soln_L1_svs == 30
-    assert bestpos.num_soln_multi_svs == 30
-    assert bestpos.extended_solution_status2 == 0x00
-    assert bestpos.ext_sol_stat == 0x06
-    assert bestpos.gal_and_bds_mask == 0x39
-    assert bestpos.gps_and_glo_mask == 0x33
+    assert isinstance(message, BESTPOS)
+    assert message.solution_status == ne.enums.SolStatus.SOL_COMPUTED
+    assert message.position_type == ne.enums.SolType.SINGLE
```

Comment:
```suggestion
    assert message.solution_status == SolStatus.SOL_COMPUTED
    assert message.position_type == SolType.SINGLE
```
Can make these and other checks more concise now that `from novatel_edie.enums import *` works.